### PR TITLE
ENH: Updating for ACT V2.0 release

### PR DESCRIPTION
**Pull request by AdamTheisen**
Don't merge until ACT V2.0 is released.

### CODE REVIEW

File: radtraq/tests/test_proc.py [hunk 1]
```diff
@@ -73,14 +73,14 @@ def test_extract_profile_at_lat_lon():
     profile_obj = radtraq.proc.profile.extract_profile_at_lat_lon(
         obj, 29.68, -95.08, variables=variables
     )
-
+
```
Comment (zssherman) [inline radtraq/tests/test_proc.py:76]: Added whitespace needs removal.

**Comment by zssherman** [comment]
@AdamTheisen I left one comment, but the error is a missing paranthesis

**Comment by zssherman** [comment]
@AdamTheisen ruff changes look good, think just the whitespace mentioned above I think is still there, but besides that think this should be good to merge.

**Comment by AdamTheisen** [comment]
@zssherman I'm looking at my code and I don't see that space anywhere.  Going to merge away once approved

**Review by zssherman — APPROVED** [review]

**Comment by zssherman** [comment]
@AdamTheisen Gotcha, yeah could have been my browser wasn't updating to the changes, yeah feel free to merge!

**Pull request merged**